src/components/common/CookieBanner/styles.module.css

```diff
@@ -3,7 +3,7 @@
   position: fixed;
   z-index: 1300;
   bottom: var(--space-2);
-  left: var(--space-2);
+  right: var(--space-2);
```

Comment:
I suggest to also change the responsive style from left to right further down in this file